docs/concepts/controllers/statefulsets.md

```diff
@@ -0,0 +1,171 @@
+---
+assignees:
+- bprashanth
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
+---
+
+{% capture overview %}
+**StatefulSets are a beta feature in 1.5. This feature replaces the 
+Pet Sets feature from 1.4. Users of Pet Sets are referred to the 1.5 
+[Upgrade Guide](/docs/tasks/stateful-set/upgrade-from-petsets-to-stateful-sets/)
+for further information on how to upgrade existing Pet Sets to StatefulSets.**
+
+A StatefulSet is a Controller that provides a unique identity to its Pods. It provides
+guarantees about the ordering of deployment and scaling.
+{% endcapture %}
+
+{% capture body %}
+
+### When to Use a StatefulSet
+StatefulSets are valuable for applications that require one or more of the 
+following.
+
+* Stable, unique network identifiers.
+* Stable, persistent storage.
+* Ordered, graceful deployment and scaling.
+* Ordered, graceful deletion and termination.
+
+In the above, by stable, we mean persistent across Pod (re) schedulings.
+As it is generally easier to manage, if an application doesn't require any of 
```

Comment:
"If an application doesn't require any stable identifiers or ordered deployment, deletion, or scaling, you should deploy your application with a controller that provides a set of stateless replicas. Such controllers, such as [Deployment] or [ReplicaSet], are generally easier to manage and may be better suited to your needs."